types/jasmine/jasmine-tests.ts

```diff
@@ -1479,6 +1500,39 @@ describe("createSpyObj", function() {
             jasmine.createSpyObj('BaseName', {});
         }).toThrow("createSpyObj requires a non-empty array or object of method names to create spies for");
     });
+
+    it("creates an object with spy properties if a second list is passed", function() {
+        const spyObj = jasmine.createSpyObj('base', ['method1'], ['prop1']);
+
+        expect(spyObj).toEqual({
+          method1: jasmine.any(Function)
+        });
+
+        expect(spyObj.prop1).toBeUndefined();
```

Comment:
Not sure this test adds any value, as it would also pass if you simply left out the second list